### PR TITLE
Fix destination integration tests

### PR DESCRIPTION
**Pull request by grishick**
Fixing integration tests for:
 - destination-mssql
 - destination-postgres
 - destination-postgres-strict-encrypt

Fixing some of the integration tests for:
 - destination-mysql
 - destination-clickhouse

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@
     normalizationRepository: airbyte/normalization-clickhouse
     normalizationTag: 0.4.0
     normalizationIntegrationType: clickhouse
-  supportsDbt: true
+  supportsDbt: false
```
Comment (ryankfu) [inline airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:96]: did clickhouse not support `dbt` or are we turning off normalization for clickhouse here?

File: airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java [hunk 2]
```diff
@@ -130,7 +130,8 @@ private String getImageNameWithoutTag() {
     return getImageName().contains(":") ? getImageName().split(":")[0] : getImageName();
   }
 
-  private Optional<StandardDestinationDefinition> getOptionalDestinationDefinitionFromProvider(final String imageNameWithoutTag) {
+  protected Optional<StandardDestinationDefinition> getOptionalDestinationDefinitionFromProvider(
+                                                                                                 final String imageNameWithoutTag) {
```
Comment (ryankfu) [inline airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java:134]: Nit: seems weird this got moved to another line

**Review by ryankfu — APPROVED** [review]
LGTM, minor nit but better to unblock integration tests

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml (hunk 1 above)
Comment (grishick) [reply airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:96]: Clickhouse does not actually support custom `dbt` transformations and clickhouse `spec.json` has `supportsDbt` set to `false`. So there was a discrepancy between `destination_definitions.yaml `and `spec.json`. My guess is this happened with normalization configs were moved to `destination_definitions.yaml` via a script (@evantahler is this a fair assumption?)

**Review by davinchia — COMMENTED** [review]
Looks good! I just came out of a meeting where we were discussed being able to note incremental normalisation support per connection in the test - looks like you already got there!

**Pull request merged**